test/test_mimic_explainer.py

```diff
@@ -295,6 +295,15 @@ def test_dataframe_model(self, mimic_explainer, sample_cnt_per_grain, grains_dic
             kwargs['categorical_features'] = ['fruit']
         mimic_explainer(model, X, LGBMExplainableModel, features=features, model_task=model_task, **kwargs)
 
+    @pytest.mark.parametrize('if_predictions_as_dataframe', [True, False])
+    def test_explain_model_binary_classification_with_different_format_predictions(
+            self, mimic_explainer, if_predictions_as_dataframe):
+        x_train, y_train, X_test, y_test, classes = create_binary_classification_dataset()
+        model = PredictAsDataFrameTestModel(return_predictions_as_dataframe=if_predictions_as_dataframe)
+        model.fit(x_train, y_train)
+        kwargs = {'explainable_model_args': {'n_jobs': 1}, 'augment_data': False, 'reset_index': True}
+        mimic_explainer(model, x_train, LGBMExplainableModel, **kwargs)
```

Comment:
LightGBM handles a lot more types than scikit-learn usually - I would just loop over all surrogate model types, since I'm not sure if the scikit-learn ones can actually handle the labels column as a dataframe -- unless we decide to convert it to numpy which I think might be more manageable

Reply:
Done.